### PR TITLE
Hqx fix

### PR DESCRIPTION
**Pull request by jakearchibald**
No longer canes the CPU in Chrome.

### CODE REVIEW

**Comment by jakearchibald** [comment]
@surma +1's via chat

File: codecs/hqx/build.sh [hunk 1]
```diff
@@ -9,6 +9,9 @@ echo "============================================="
   rustup run nightly \
     wasm-pack build --target no-modules
   wasm-strip pkg/squooshhqx_bg.wasm
+  echo "Optimising WASM so it doesn't break Chrome (this takes like 10-15mins. get a cup of tea)"
```
Comment (mathiasbynens) [inline codecs/hqx/build.sh]: nit: `/s/WASM/Wasm/`

Comment (jakearchibald) [reply codecs/hqx/build.sh]: Fixed in 2f834db22499af1b7ff1e7c050a52b9a0d0e2633

Comment (mathiasbynens) [reply codecs/hqx/build.sh]: that’s still wrong lolol

it’s supposed to be “Wasm” with (only) the “W” capitalized

Comment (mathiasbynens) [reply codecs/hqx/build.sh]: https://github.com/GoogleChromeLabs/squoosh/pull/630

**Pull request merged**